site/docs/tools/index.md

```diff
@@ -20,16 +20,16 @@ the following:
 
 ## Feature Matrix
 
-| |Spark 2 [^1]|Spark 3[^2]|[Nessie CLI](cli.md)|
-| --- | --- | --- | --- |
-|Read Default Branch|:material-check:| :material-check: | |
-|Read Any Branch/Tag/Hash|:material-check:| :material-check: | |
-|Write Default Branch|:material-check:| :material-check: | |
-|Write Any Branch/Tag/Hash|:material-check:| :material-check: | |
-|Create Branch| :material-check: | :material-check: | :material-check:|
-|Create Tag| :material-check: | :material-check: |:material-check:|
-|Iceberg Tables|:material-check:|:material-check:| |
-|Delta Lake Tables||:material-check:| |
+| |Spark 2 [^1]|Spark 3[^2]|[Nessie CLI](cli.md)|Flink|
```

Comment:
I know this PR is about Flink, but Hive part is missing too. can we add it here ? so that feature matrix will be complete ?

Reply:
hive stuff from the feature matrix was removed as part of https://github.com/projectnessie/nessie/commit/e8f451f8722b0c9e7ccf97c21eb10d523b699062. Let's tackle this separately